sw/device/tests/sim_dv/i2c_device_tx_rx_test.c

```diff
@@ -110,47 +116,141 @@ void check_addr(uint8_t addr, dif_i2c_id_t id0, dif_i2c_id_t id1) {
         ((addr & id1.mask) == id1.address));
 }
 
+// handle i2c index related configure
+void config_i2c_with_index(void) {
+  uint8_t i = 0;
+  switch (kI2cIdx) {
+    case 0:
+      i2c_base_addr = TOP_EARLGREY_I2C0_BASE_ADDR;
+      i2c_irq_fmt_watermark_id = kTopEarlgreyPlicIrqIdI2c0FmtWatermark;
```

Comment:
Please do the same for the other constants, see also @andreaskurth 's comment.

Reply:
This is changed in the latest version.
Most of these constants are now in the function `i2c_testutils_pinmux` in `12c_testutils.c`